schema/schema.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE music_albums(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  on_spotify BOOLEAN,
+  FOREIGN KEY(id) REFERENCES genres(id),
+  publish_date DATE,
+  archived BOOLEAN
+);
+
+CREATE TABLE genres(
+  id INT GENERATED ALWAYS AS IDENTITY,
+  name VARCHAR(255)
+  PRIMARY KEY(id)
+);
```

Comment:
- [ ] I have realized that you did not implement all the project requirements for the database. I can only see two tables and there are supposed to be SQL commands that create the :  1. books  2. music_albums  3. games  4. authors  5. labels  6. genres. These are the total number of tables with all their columns that are supposed to be created. Kindy ensure that you add them all.

![image](https://user-images.githubusercontent.com/86318284/197239073-694b95c3-9963-4405-9e2d-7a4b2746040c.png)

Reply:
All the database files are implemented in a different schema file. But, we will merge it into one schema file.